stdlib/scripts/run-tests.sh

```diff
@@ -23,6 +23,12 @@ mkdir -p "${BUILD_DIR}"
 
 source "${SCRIPT_DIR}"/build-stdlib.sh
 
+echo "Validating API documentation strings in the Standard Library."
+mojo doc validate "${REPO_ROOT}/stdlib/src" || {
```

Comment:
Same here, `mojo doc validate` doesn't exist. It's `mojo doc`, this is why the CI Doesn't understand what is going on (but I think the error message is not correct, `mojo doc` should output that the directory `validate` doesn't exist.)